### PR TITLE
Style the Calculator #5

### PR DESCRIPTION
**Pull request by sondos-zaro**
design the calculator that contains :
1.title
2.input
2.button for number
3.button for operations
4.button for currency

image

### CODE REVIEW

**Comment by someyaaltous** [comment]
Nice work Sondos, but I have some notes:

- The calculator should be in center of the page 

- The display should not be an input, change this 
```
<input type="text" id="input-p">
```
to this 

```
<div id="input-p">0</div>
```

**Review by someyaaltous — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (Rahaf-96) [inline .vscode/settings.json]: delete this folder `.vscode`, we don't need it here

**Comment by saharAdem** [comment]
Great Job!. But replace `in-progress` label with `in-review`  label for your pull request when you are waiting for a review.

**Review by saharAdem — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
Great job :heart_eyes: keep it up :rocket:

**Pull request merged**